### ISSUE TITLE
Credentials should have a .revoke() method.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#98]
```
The .revoke() method should programatically revoke the token:

  http://code.google.com/apis/accounts/docs/OAuth2WebServer.html#tokenrevoke

And then call .delete() on the underlying storage.
```

Original issue reported on code.google.com by `jcgregorio@google.com` on 6 Feb 2012 at 7:48

**Comment by GoogleCodeExporter** [comment]
```
Issue 97 has been merged into this issue.
```

Original comment by `jcgregorio@google.com` on 6 Feb 2012 at 7:49

**Comment by GoogleCodeExporter** [comment]
```
Issue 229 has been merged into this issue.
```

Original comment by `jcgregorio@google.com` on 3 Jan 2013 at 12:26

**Comment by GoogleCodeExporter** [comment]
```
Joe,

You're fine with me owning this?
```

Original comment by `dhermes@google.com` on 3 Jan 2013 at 12:28

**Comment by GoogleCodeExporter** [comment]
```
Yes, take it away :)
```

Original comment by `jcgregorio@google.com` on 3 Jan 2013 at 2:16

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision f5ce7e5b795a.
```

Original comment by `dhermes@google.com` on 6 Feb 2013 at 5:19
- Changed state: **Fixed**
